client/src/App.js

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './app.css';
+import { Dashboard, Home } from './layouts';
+import { LoginPage, RegisterPage, LoginGymPage, GymPage, NotfoundPage } from './pages';
```

Comment:
```suggestion
import { LoginPage, RegisterPage, LoginGymPage, GymPage, NotfoundPage } from './Pages';
```
Linux is a case-sensitive in files naming, So take care about this